### ISSUE TITLE
[PRE REVIEW]: stabm: Stability Measures for Feature Selection

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2900]
**Submitting author:** @bommert (<a href="http://orcid.org/0000-0002-1005-9351">Andrea Bommert</a>)
**Repository:** <a href="https://github.com/bommert/stabm" target ="_blank">https://github.com/bommert/stabm</a>
**Version:** 1.1.4
**Editor:** @marcosvital
**Reviewers:** @mcavs, @clauswilke, @osorensen
**Managing EiC:** Kevin M. Moerman

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**Author instructions**

Thanks for submitting your paper to JOSS @bommert. **Currently, there isn't an JOSS editor assigned** to your paper.

@bommert if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.03 s (912.8 files/s, 101146.6 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                               19            299            508           1532
TeX                              2             28              0            261
Markdown                         2             21              0            113
YAML                             2             23              0             95
Rmd                              2             29             65             18
-------------------------------------------------------------------------------
SUM:                            27            400            573           2019
-------------------------------------------------------------------------------


Statistical information for the repository '794102bdbaa9b83685516ce9' was
gathered on 2020/12/11.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: :point_left:

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1155/2017/7907163 is OK
- 10.1016/j.csda.2019.106839 is OK
- 10.1109/IRI.2012.6303031 is OK
- 10.1007/978-3-319-25226-1_7 is OK
- 10.5169/seals-266450 is OK
- 10.1093/bioinformatics/btl400 is OK
- 10.1109/icdmw.2009.32 is OK
- 10.2331/suisan.22.531 is OK
- 10.1007/978-3-319-46227-1_28 is OK
- 10.1007/978-3-030-46150-8_20 is OK
- 10.1109/tpami.2010.34 is OK
- 10.1109/ictai.2013.63 is OK
- 10.1109/tcbb.2011.47 is OK
- 10.2202/1544-6115.1307 is OK
- 10.18637/jss.v031.i02 is OK
- 10.1016/j.softx.2020.100526 is OK

MISSING DOIs

- 10.2307/1932409 may be a valid DOI for title: Measures of the Amount of Ecologic Association Between Species

INVALID DOIs

- 10.1007/s10115-006-0040-8 is INVALID
```

**Comment by Kevin-Mattheus-Moerman** [comment]
@marcosvital you were suggested as editor for this work. Can you review if this is in scope (you can trigger a board scope query if needed), and if so handle this submission? Thanks

**Comment by marcosvital** [comment]
Yes, I can do that, @Kevin-Mattheus-Moerman!

@bommert, I'll handle your submission. The first thing I am going to do is to trigger a scope query, so other editors can help me check if you submission falls into JOSS requirements.

**Comment by marcosvital** [comment]
@whedon query scope

**Comment by whedon** [comment]
Submission flagged for editorial review.

**Comment by marcosvital** [comment]
@whedon assign @marcosvital as editor

**Comment by whedon** [comment]
OK, the editor is @marcosvital

**Comment by danielskatz** [comment]
@marcosvital - going forward, if you can refrain from becoming the editor of papers undergoing scope discussion, it would make it easier for AEiCs to track and act on the scope discussion

**Comment by danielskatz** [comment]
Sorry @marcosvital,  now I see that @Kevin-Mattheus-Moerman asked you to handle it this way, which I think is problematic

@Kevin-Mattheus-Moerman, I think it's the job of the AEiC to try to handle to initial scope query if needed before passing it onto an editor...

**Comment by Kevin-Mattheus-Moerman** [comment]
> @Kevin-Mattheus-Moerman, I think it's the job of the AEiC to try to handle to initial scope query if needed before passing it onto an editor...

@danielskatz yes apologies for deviating from protocol.

**Comment by marcosvital** [comment]
No problem! I'll wait until the scope is resolved, and can handle the submission if is considered to be on scope.

**Comment by arfon** [comment]
@marcosvital - I think this is ok to proceed to review.

**Comment by marcosvital** [comment]
Hi, @bommert , I will be the editor of your submission.

Please take a look at the first manuscript proof generated above, and check if everything seems to be ok with it. If you make any changes, you can use `@whedon generate pdf` to have whedon create an update version (or just let me know and I'll do it).

At the same time, we can start assigning reviewers for you submission. We usually ask the authors for suggestions, so let me know if you have anyone in mind that could do this - you can paste the github profile of anyone you think that could do it (but please don't tag anyone yet).
You can also check [this list of people](https://bit.ly/joss-reviewers) that have agreed to review for JOSS, and see if any of them may be suitable to review this submission.

**Comment by bommert** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: :point_left:

**Comment by bommert** [comment]
Thank you for being the editor of our submission!

The manuscript looks good.

From the list of people that have agreed to review for JOSS, the following persons seem to have expertise in the field of our contribution:
road2stat, martinju, prdm0, nhejazi, kdpeterson51, mcavs, osorensen, robohyun66, clauswilke.

**Comment by marcosvital** [comment]
Hi, @bommert, thank you for the suggestions. I'll start to contact some of the possible reviewers soon.

In the meantime, do you think you could wok on your summary in order to include a brief description of the methods for a wider audience? As stated at [JOSS documentantion](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain), the paper should include:

> A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience.

**Comment by marcosvital** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/s10115-006-0040-8 is OK
- 10.1155/2017/7907163 is OK
- 10.1007/978-3-030-64583-0_19 is OK
- 10.1016/j.csda.2019.106839 is OK
- 10.1109/IRI.2012.6303031 is OK
- 10.1007/978-3-319-25226-1_7 is OK
- 10.5169/seals-266450 is OK
- 10.1093/bioinformatics/btl400 is OK
- 10.1109/icdmw.2009.32 is OK
- 10.2331/suisan.22.531 is OK
- 10.1007/978-3-319-46227-1_28 is OK
- 10.1007/978-3-030-46150-8_20 is OK
- 10.1109/tpami.2010.34 is OK
- 10.1109/ictai.2013.63 is OK
- 10.1109/tcbb.2011.47 is OK
- 10.2202/1544-6115.1307 is OK
- 10.18637/jss.v031.i02 is OK
- 10.1016/j.softx.2020.100526 is OK

MISSING DOIs

- 10.2307/1932409 may be a valid DOI for title: Measures of the Amount of Ecologic Association Between Species

INVALID DOIs

- None
```

**Comment by marcosvital** [comment]
@bommert, please also look into the possible missing DOI above, ok?

**Comment by marcosvital** [comment]
Dear @nhejazi, @prdm0 and @mcavs, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by mcavs** [comment]
Dear @marcosvital, I would!

**Comment by bommert** [comment]
@whedon generate pdf
@whedon check references

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: :point_left:

**Comment by nhejazi** [comment]
Unfortunately I'm unable to provide a review at this time due to a confluence of deadlines.

**Comment by bommert** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/s10115-006-0040-8 is OK
- 10.1155/2017/7907163 is OK
- 10.1007/978-3-030-64583-0_19 is OK
- 10.1016/j.csda.2019.106839 is OK
- 10.1109/IRI.2012.6303031 is OK
- 10.1007/978-3-319-25226-1_7 is OK
- 10.5169/seals-266450 is OK
- 10.2307/1932409 is OK
- 10.1093/bioinformatics/btl400 is OK
- 10.1109/icdmw.2009.32 is OK
- 10.2331/suisan.22.531 is OK
- 10.1007/978-3-319-46227-1_28 is OK
- 10.1007/978-3-030-46150-8_20 is OK
- 10.1109/tpami.2010.34 is OK
- 10.1109/ictai.2013.63 is OK
- 10.1109/tcbb.2011.47 is OK
- 10.2202/1544-6115.1307 is OK
- 10.18637/jss.v031.i02 is OK
- 10.1016/j.softx.2020.100526 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by bommert** [comment]
@marcosvital We have added an explanatory example in the beginning of our summary section. Do you think that this makes the high-level description of our methods understandable better for a wider audience?

**Comment by marcosvital** [comment]
Thank you, @mcavs! We will wait as we gather one or two more reviewers before starting the review process, ok?

No problem, @nhejazi, and thanks for the quick reply.

**Comment by marcosvital** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02900/joss.02900/10.21105.joss.02900.pdf) :page_facing_up: :point_left:

**Comment by marcosvital** [comment]
> 
> 
> @marcosvital We have added an explanatory example in the beginning of our summary section. Do you think that this makes the high-level description of our methods understandable better for a wider audience?

Yes, seems good to me, @bommert!

**Comment by marcosvital** [comment]
@bommert, as we wait for reviewers, please take a look at some of JOSS requirements for the package's documentation: https://joss.readthedocs.io/en/latest/review_checklist.html#documentation

Specifically, I think you'll need to add some information at the repository related to the community guidelines:

> Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

**Comment by marcosvital** [comment]
Dear @clauswilke, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by clauswilke** [comment]
Sure.

**Comment by bommert** [comment]
> 
> 
> @bommert, as we wait for reviewers, please take a look at some of JOSS requirements for the package's documentation: https://joss.readthedocs.io/en/latest/review_checklist.html#documentation
> 
> Specifically, I think you'll need to add some information at the repository related to the community guidelines:
> 
> > Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

We have added the information in the readme file.

**Comment by marcosvital** [comment]
Thank you, @bommert.

@clauswilke and @mcavs, thank you for accepting to review this submission. I'm trying to reach a third reviewer so we can the reviewing processes soon, ok?

**Comment by marcosvital** [comment]
Dear @osorensen, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by osorensen** [comment]
@marcosvital: yes

**Comment by marcosvital** [comment]
Thank you, @osorensen!

To all: we will now move on to a new issue to start reviewing.

**Comment by marcosvital** [comment]
@whedon assign @mcavs as reviewer

**Comment by whedon** [comment]
OK, @mcavs is now a reviewer

**Comment by marcosvital** [comment]
@whedon add @clauswilke as reviewer

**Comment by whedon** [comment]
OK, @clauswilke is now a reviewer

**Comment by marcosvital** [comment]
@whedon add @osorensen as reviewer

**Comment by whedon** [comment]
OK, @osorensen is now a reviewer

**Comment by marcosvital** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/3010.
